src/server.rs

```diff
@@ -234,6 +235,16 @@ async fn index(
     mut payload: web::Payload,
     req: HttpRequest,
 ) -> impl Responder {
+    let mut queries = HashMap::new();
+    
+    if req.query_string().len() > 0 {
+        let split = req.query_string().split("&");
+        for s in split {
+            let params = s.split_once("=").unwrap_or((s, ""));
+            queries.insert(params.0, params.1);
+        }
+    }
+
```

Comment:
Do we want the queries hashmap to be available in every HTTP request or just the GET requests methods?

I usually use this kind of params in GET requests only. But I am not sure about the standard of WEB requests.

Reply:
Personally, I think it's generally only necessary for GET requests. That being said, this project doesn't exist for just me. I expect developers will want to use features like this in unorthodox ways. There didn't seem to be a good reason to restrict queries solely to GET requests, so I left it open to all types.

Reply:
> I expect developers will want to use features like this in unorthodox ways

Absolutely! Robyn is based on making unorthodox tradeoffs for better performance. I love it. :D